students/julienjass/Lesson04/Trigram.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import random
+import sys
+
+def extract_words(filename):
+    with open(filename) as infile:
+        words = []
+        for line in infile:
+            if line.startswith("End of the Project Gutenberg"):
+                break
+            for char in ['/', '\\', '.', ',', '(', ')', ':', ';', '"', '?', '!', '-', '\'']:
+                line = line.replace(char, '')
+            words.extend(line.split())
+    return words
+
+def build_trigram(words):
+    trig = {}
+    for i in range(len(words) - 2):
+        first = words[i]
+        second = words[i + 1]
+        third = words[i + 2]
+        pair = (first, second)
+        trig.setdefault(pair, []).append(third)
+    return trig
+
+def create_new_text(trigram):
+    pair = random.choice(list(trigram.keys()))
+    sentence = []
+    sentence.extend(pair)
+    for i in range(30):
+        followers = trigram[pair]
+        sentence.append(random.choice(followers))
+        pair = tuple(sentence[-2:])
+    result = ' '.join(sentence).capitalize().replace(' i ', ' I ')+"."
+    return result
```

Comment:
Since this produces a single 30 word sentence, can you modify the code (either here or in the `__if__ name == '__main__'` block) to produce a couple hundred words of output per the assignment instructions?

Reply:
I see. Thanks. will modify it and resubmit it with my next PR tonight.